vm/embedded/tests/htlc_test.go

```diff
@@ -0,0 +1,694 @@
+package tests
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/zenon-network/go-zenon/common/types"
+	"github.com/zenon-network/go-zenon/rpc/api/embedded"
+	"github.com/zenon-network/go-zenon/vm/constants"
+	"github.com/zenon-network/go-zenon/vm/embedded/definition"
+
+	g "github.com/zenon-network/go-zenon/chain/genesis/mock"
+	"github.com/zenon-network/go-zenon/chain/nom"
+	"github.com/zenon-network/go-zenon/common"
+	"github.com/zenon-network/go-zenon/common/crypto"
+	"github.com/zenon-network/go-zenon/zenon/mock"
+)
+
+const (
+
+	// why not just make this pubic in mock genesis?
```

Comment:
lol typo

Reply:
Contrary to popular alien culture, there is nothing pubic about a Pillar and you should not be mocking the genesis block.